test-suite-geb/build.gradle.kts

```diff
@@ -24,9 +24,7 @@ dependencies {
     testImplementation(projects.micronautSecuritySession)
     testImplementation(libs.geb.spock)
 
-    testImplementation(platform(libs.testcontainers.bom))
     testImplementation(libs.testcontainers.selenium)
-    testImplementation(libs.testcontainers)
```

Comment:
don't need to set `libs.testcontainers`, because you already use `libs.testcontainers.selenium`